src/util/alloc/bumpallocator.rs

```diff
@@ -82,7 +85,7 @@ impl<VM: VMBinding> Allocator<VM> for BumpAllocator<VM> {
     // Performs three kinds of allocations: (i) if the hard limit has been met;
```

Comment:
Could you update the commends to say "precise" stress here as well?

Reply:
Done. Thanks for the review.